modules/memberships/class-jetpack-memberships.php

```diff
@@ -270,12 +251,13 @@ public function render_button( $attrs ) {
 		$button_styles = implode( ';', $button_styles );
 		add_thickbox();
 		return sprintf(
-			'<button data-blog-id="%d" data-powered-text="%s" data-plan-id="%d" data-lang="%s" class="%s" style="%s">%s</button>',
+			'<div class="wp-block-button %s"><a role="button" data-blog-id="%d" data-powered-text="%s" data-plan-id="%d" data-lang="%s" class="wp-block-button__link %s" style="%s">%s</a></div>',
```

Comment:
Yes, I wasn't sure whether we should do this or not. I didn't know if there was something relying on this being rendered as a `button` not wrapped in a `div`. I suppose the answer is that it shouldn't be!